catalyst/dl/experiment/gan.py

```diff
@@ -37,7 +37,7 @@ def __init__(
         distributed_params: Dict = None,
         monitoring_params: Dict = None,
         initial_seed: int = 42,
-        phase2callbacks: Dict[str, List[str]] = None,
+        phase2callbacks: Dict[str, List[str]] = {},
```

Comment:
it's better to use None and `phase2callbacks  or {}` after that

Reply:
Done